metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -0,0 +1,83 @@
+from os import listdir
+from os.path import isfile, join
+from subprocess import run, PIPE
+
+import kfp
+
+import pytest
+
+"""
+From the folder `tests`, run: 
+`python -m pytest -s -n 3 run_integration_tests.py`
+
+This script runs all the flows in the `sample_flows` directory. It creates
+each kfp run, waits for the run to fully complete, and prints whether
+or not the run was successful. It also checks to make sure the logging
+functionality works.
+
+More specifically, the tests spawn KFP runs and ensure the spawning processes
+have a returncode of 0. However, the logs of the runs within KFP are simply logged
+to the screen, and it is the responsibility of the user to read these logs to ensure
+the run executed correctly on KFP.
+
+Parameters:
+-n: specifies the number of parallel processes used by PyTest.
+
+Sometimes, the tests may fail on KFP due to resource quota issues. If they do,
+try reducing -n (number of parallel processes) so less simulateneous
+KFP will be scheduled.
+
+"""
+
+
+def parse_run_id(output):
+    start_idx = output.find("run_id|")
+    end_idx = output.find("|end_id")
+
+    if start_idx == -1 or end_idx == -1:
+        return -1
+
+    run_id = output[start_idx + len("run_id|") : end_idx]
+    return run_id
+
+
+def obtain_flow_file_paths(flow_dir_path):
+    file_paths = [
+        f
+        for f in listdir(flow_dir_path)
+        if isfile(join(flow_dir_path, f)) and not f.startswith(".")
```

Comment:
The `not f.startswith(".")` removes hidden files (like `.DS_Store`), which cause the tester to show an error and clutter the screen for the viewer.